src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -3886,7 +3898,7 @@ void gfx_run(Gfx* commands, const std::unordered_map<Mtx*, MtxF>& mtx_replacemen
                                             !game_renders_to_framebuffer);
     gfx_rapi->start_frame();
     gfx_rapi->start_draw_to_framebuffer(game_renders_to_framebuffer ? game_framebuffer : 0,
-                                        (float)gfx_current_dimensions.height / gfx_native_dimensions.height);
+                                        gfx_calculate_noise_scale(), g_rdp.alpha_test_value);
```

Comment:
Passing in `alpha_test_value` through `start_draw_to_framebuffer` means that the value is only read and set once at the very beginning of the frame.

But you implemented `gfx_dp_set_blend_color` which influences this value, but it'll never be updated to the shader this way.